src/domain/worker.rs

```diff
@@ -0,0 +1,36 @@
+use std::fmt::Display;
+
+use super::error::{LoadBalancerError, LoadBalancerResult};
+
+#[derive(Debug, PartialEq, Clone)]
+pub struct WorkersList(Vec<String>);
+
+impl WorkersList {
+    pub fn parse(list: Vec<String>) -> LoadBalancerResult<Self> {
+        if list.is_empty() {
+            return Err(LoadBalancerError::ConfigError(
+                "At least 1 worker must be specified.".into(),
+            ));
+        }
+
+        Ok(Self(list))
+    }
```

Comment:
Great use of the "parse don't validate" pattern! 🔥 
You could additionally validate that the strings themselves are properly formatted.